### ISSUE TITLE
[Docs] 404/Broken links on adapters page 

### ISSUE THREAD

**Issue by Rohith-Raju** [meshery/meshery#5517]
#### Current State
Site Link : https://docs.meshery.io/extensibility/adapters
![screen-recorder-sun-may-15-2022-09-33-59](https://user-images.githubusercontent.com/55309324/168456755-125a64b1-6043-411f-afe3-c5b1752110ec.gif)
#### Desired State

Should not be broken and take the user to the desired page.

---

#### [Contributor Guides](https://docs.meshery.io/project/contributing) and Resources
- 📚 [Instructions for contributing to documentation](https://docs.meshery.io/project/contributing/contributing-docs)
   - Meshery documentation [site](https://docs.meshery.io/) and [source](https://github.com/meshery/meshery/tree/master/docs)
- 🛠 [Meshery Build & Release Strategy](https://docs.meshery.io/project/build-and-release)
- 🎨 Wireframes and designs for Meshery UI in [Figma](https://www.figma.com/file/SMP3zxOjZztdOLtgN4dS2W/Meshery-UI)
- 🙋🏾🙋🏼 Questions: [Layer5 Discussion Forum](https://discuss.layer5.io) and [Layer5 Community Slack](http://slack.layer5.io)

**Comment by leecalcote** [comment]
Thanks, @Rohith-Raju. Is this an item that you would  like to work on?

**Comment by Rohith-Raju** [comment]
Sure, would love to @leecalcote

**Comment by Rohith-Raju** [comment]
Hey so I went through the code and this is what I found
![image](https://user-images.githubusercontent.com/55309324/168457919-d9c05bcd-ea43-4b48-8d15-6b6306b7604f.png)
Here the links point to each of its files in the directory. 
![image](https://user-images.githubusercontent.com/55309324/168457934-252a83e6-a16b-44ae-b2f5-d981a1c49535.png)
But there are no files that could be found with that name which explains the page not found.

What are my next steps?
